web/src/admin/blueprints/BlueprintForm.ts

```diff
@@ -138,33 +139,39 @@ export class BlueprintForm extends ModelForm<BlueprintInstance, string> {
                               >
                               </ak-search-select>
                           </ak-form-element-horizontal>`
-                        : html``}
+                        : nothing}
                     ${this.source === blueprintSource.oci
-                        ? html`<ak-form-element-horizontal label=${msg("URL")} name="path">
-                              <input
-                                  type="text"
-                                  value="${ifDefined(this.instance?.path)}"
-                                  class="pf-c-form-control"
-                                  required
-                              />
-                              <p class="pf-c-form__helper-text">
-                                  ${msg(
-                                      "OCI URL, in the format of oci://registry.domain.tld/path/to/manifest.",
-                                  )}
-                              </p>
-                              <p class="pf-c-form__helper-text">
-                                  ${msg("See more about OCI support here:")}&nbsp;
-                                  <a
-                                      target="_blank"
-                                      rel="noopener noreferrer"
-                                      href="${docLink(
-                                          "/docs/customize/blueprints/?utm_source=authentik#storage---oci",
-                                      )}"
-                                      >${msg("Documentation")}</a
-                                  >
-                              </p>
-                          </ak-form-element-horizontal>`
-                        : html``}
+                        ? html` <ak-text-input
+                              name="path"
+                              label=${msg("OCI URL")}
+                              input-hint="code"
+                              required
+                              placeholder="oci://..."
+                              value="${ifDefined(this.instance?.path)}"
+                              .bighelp=${html`<p class="pf-c-form__helper-text">
+                                      ${msg(
+                                          html` A valid OCI manifest URL, prefixed with the protocol
+                                              e.g.&nbsp;<code
```

Comment:
Some words such as `e.g.` seem to have a special effect on VoiceOver and are automatically substituted for a phrase:

"e.g. foo, bar, baz." -> "Such as<breath>, foo, bar, baz."

Maybe we can put together a list as they're discovered.